processors/include/SvtRawDataAnaProcessor.h

```diff
@@ -0,0 +1,101 @@
+#ifndef __RAWSVTHIT_ANAPROCESSOR_H__
+#define __RAWSVTHIT_ANAPROCESSOR_H__
+
+//HPSTR
+#include "HpsEvent.h"
+#include "RawSvtHit.h"
+#include "RawSvtHitHistos.h"
+#include "AnaHelpers.h"
+#include "Event.h"
+#include "BaseSelector.h"
```

Comment:
So you don't use BaseSelectorSVT? Why did you add that class? Probably remove it, looks the same as BaseSelector to me anyway. You can always just inherit from BaseSelector.

Reply:
Removed.